client-java/controller/src/main/java/org/evomaster/client/java/controller/internal/EMController.java

```diff
@@ -223,11 +223,24 @@ public Response getSutInfo(@Context HttpServletRequest httpServletRequest) {
                         dto.rpcProblem.localAuthEndpoints.add(e.getValue().getDto());
                     }
                 }
-                // handled seeded tests
-                dto.rpcProblem.seededTestDtos = noKillSwitch(() -> sutController.handleSeededTests());
 
-                if (dto.isSutRunning){
-                    noKillSwitch(() -> sutController.getSeededExternalServiceResponseDto());
+                try{
+                    // handled seeded tests
+                    dto.rpcProblem.seededTestDtos = noKillSwitch(() -> sutController.handleSeededTests());
+
+                    if (dto.isSutRunning){
+                        noKillSwitch(() -> sutController.getSeededExternalServiceResponseDto());
+                    }
+                }catch (RuntimeException e){
+                    StringBuilder msg = new StringBuilder("Fail to handle specified seeded tests " + e.getMessage());
+                    if (e.getStackTrace() != null && e.getStackTrace().length > 0){
```

Comment:
you are calling `getStackTrace()` 4 times. note that each call make a copy... maybe best to call it once, and save it in variable